common/src/main/java/net/neoforged/gradle/common/runtime/tasks/action/DownloadFileAction.java

```diff
@@ -106,30 +106,25 @@ public void execute() {
         }
     }
 
-    private static final class Monitor implements CopyProgressListener {
-        private final GradleInternalUtils.ProgressLoggerWrapper progress;
-
-        private Monitor(GradleInternalUtils.ProgressLoggerWrapper progress) {
-            this.progress = progress;
-        }
+    private record Monitor(GradleInternalUtils.ProgressLoggerWrapper progress) implements CopyProgressListener {
 
         @Override
-        public void start(CopyProgressEvent evt) {
-            progress.started();
-            progress.incrementDownloadProgress(evt.getReadBytes());
-        }
+            public void start(CopyProgressEvent evt) {
+                progress.started();
+                progress.incrementDownloadProgress(evt.getReadBytes());
+            }
 
-        @Override
-        public void progress(CopyProgressEvent evt) {
-            progress.incrementDownloadProgress(evt.getReadBytes());
-        }
+            @Override
+            public void progress(CopyProgressEvent evt) {
+                progress.incrementDownloadProgress(evt.getReadBytes());
+            }
 
-        @Override
-        public void end(CopyProgressEvent evt) {
-            progress.incrementDownloadProgress(evt.getReadBytes());
-            progress.completed();
+            @Override
+            public void end(CopyProgressEvent evt) {
+                progress.incrementDownloadProgress(evt.getReadBytes());
+                progress.completed();
+            }
         }
-    }
```

Comment:
It seems you over-indented this by one indent.

Reply:
Reformatted. Should fix it. I changed some of this to a record.